modules/kms-keyring/src/kms_keyring.ts

```diff
@@ -203,6 +206,20 @@ export function KmsKeyringClass<S extends SupportedAlgorithmSuites, Client exten
         return material
       }
 
+      /* Postcondition: A CMK must provide a valid data key.
```

Comment:
Is this the right wording if it is valid for no CMKs to produce a valid data key as long as there are no errors?